script.js

```diff
@@ -36,11 +36,12 @@ function generateQuote() {
     });
   tweetBtn.addEventListener("click", shareTweet);
 
-  shareTweet();
-
   function shareTweet(event) {
+    var quoteText = quote.innerText;
+    var quoteAuthor = author.innerText;
+
     const twitterUrl =
-      "https://twitter.com/intent/tweet?text=${quoteText} -${quoteAuthor}";
+      `https://twitter.com/intent/tweet?text=${quoteText} -${quoteAuthor}`;
```

Comment:
Changed from double quotes (`"`) to backtick as that is required for using the `${var}` syntax in JS.